docs/root/intro/life_of_a_request.rst

```diff
@@ -0,0 +1,181 @@
+.. _life_of_a_request:
+
+Life of a Request
+=================
+
+Below we follow the events that take place after the arrival of a request at the Envoy proxy from a
+downstream client, the dispatch to an upstream backend and the corresponding response path.
+
+Network topology
+----------------
+
+Envoy can be used in a wide variety of networking topologies. We focus on the inner operation
+of Envoy below, but briefly we address how Envoy relates to the rest of the network in this section.
+
+Envoy originated as a `service mesh
+<https://blog.envoyproxy.io/service-mesh-data-plane-vs-control-plane-2774e720f7fc>`_ sidecar proxy,
+factoring out load balancing, routing, observability, security and discovery services from
+applications. In the service mesh model:
+
+* Ingress listeners take requests from other nodes in the service mesh and forward them to the
```

Comment:
optional, do you think it's worth introducing downstream and upstream here
from "downstream" nodes -> to "upstream" local applications?

It took me months to remember which was "client" and which was "backend" and that was with a little diagram on my desk :-P

Reply:
Agreed, I've added a terminology section right at the start.